language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -76,6 +77,9 @@ daml2ts Options{..} pkgId pkg mbPkgName = do
 dup :: a -> (a, a)
 dup x = (x, x)
 
+dot :: T.Text -> T.Text -> T.Text
```

Comment:
```suggestion
infixr <.> -- This is the same fixity as '<>'.
(<.>) :: T.Text -> T.Text -> T.Text
```
If you defined it like this, you can write
```
a <> b <.> c <> d
```
instead of the more wordy
```
a <> (b `dot` c) <> d
```

Reply:
I did write that function but there's a conflict with `Filepath` and we use the definition from that. I could qualify that usage of course. I'll do it.